tracing_test.go

```diff
@@ -137,7 +141,7 @@ func TestStartSpan(t *testing.T) {
 		Tags: nil,
 		// TODO(tracing): Set Transaction.Data here or in
 		// Contexts.Trace, or somewhere else. Currently ignored.
-		Extra:     nil,
+		Extra:     span.Data,
```

Comment:
Does the comment above still stands if we set it now?

Reply:
Thanks for noticing, it was stale. I moved the comment over from `tracing.go` here.